x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -122,12 +123,19 @@ public void testCreateAndRestoreSearchableSnapshot() throws Exception {
         final boolean cacheEnabled = randomBoolean();
         logger.info("--> restoring index [{}] with cache [{}]", restoredIndexName, cacheEnabled ? "enabled" : "disabled");
 
+        Settings.Builder indexSettingsBuilder = Settings.builder()
+            .put(SearchableSnapshots.SNAPSHOT_CACHE_ENABLED_SETTING.getKey(), cacheEnabled)
+            .put(IndexSettings.INDEX_CHECK_ON_STARTUP.getKey(), Boolean.FALSE.toString());
+        final List<String> nonCachedExtensions;
+        if (randomBoolean()) {
+            nonCachedExtensions = randomSubsetOf(Arrays.asList("fdt", "fdx", "nvd", "dvd", "tip", "cfs", "dim"));
```

Comment:
Can we remove `.cfs `from this list?

Reply:
why?

This is just a random list to test this functionality on. What's in there is not so important. And I don't think that we are force-merging in this test, so `.cfs` could be a reality here.

Reply:
> why?

My request was mostly motivated by reducing the number of small requests executed to the fixtures as it makes debug a lot more difficult, but this test does not run on fixture so you can just forget my comment. Sorry for the noise.